src/plugins/intel_gna/ops/identity.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "identity.hpp"
+
+#include <ngraph/validation_util.hpp>
+
+#include "ngraph/attribute_visitor.hpp"
+
+namespace ov {
+namespace intel_gna {
+namespace op {
+
+Identity::Identity(const ngraph::Output<ngraph::Node>& arg) : Op({arg}) {
+    constructor_validate_and_infer_types();
+}
+
+std::shared_ptr<ngraph::Node> Identity::clone_with_new_inputs(const ngraph::OutputVector& new_args) const {
```

Comment:
```suggestion
std::shared_ptr<ngraph::Node> Identity::CloneWithNewInputs(const ngraph::OutputVector& new_args) const {
```

following google style Method should be  `CamelCase` only getter and setters should be follow naming convention of fields.

It is also related to the rest of classes

Reply:
@marcinkusm It is official OpenVINO ngraph API. Please note, that OV style is based on Google style, but has some some differencies

_OpenVINO has strict rules for naming style in public API. All classes must start with a capital letter, and methods and functions should be named in snake_case style._ 
https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/coding_style.md